### DISCUSSION TITLE
Overlay date and time

### DISCUSSION THREAD

**Discussion by yorkpeter** [gopro/labs#705] [General]
I have a GoPro Hero 10 black and I installed the GoPro Labs firmware.
I would like to superimpose on my Timelapses (videos and photos) the real time in which each frame (or photo) was taken. Same for Nightlapse.
I saw that it is very convenient to use the Android QR Control app but in none of the screens did I find anything that could be useful in obtaining what I wanted.
There is actually a "Set Date and Time" checkbox but it is used to set the date and time of the smartphone in the GoPro.
Thanks for any help you can give me.

**Comment by dnewman-gpsw** [comment]
It is there, but very manual.  Use https://gopro.github.io/labs/control/overlays/ .  Overlays are limited to video modes, no photo support, and resolutions 4k 16x9 or lower.

**Comment by yorkpeter** [comment]
Thank you. I will try this option.
However, I am sorry that it is not possible to use it for timelapse photos.
In this regard, I ask if there isn't a utility that in post-production reads the photo creation time in the EXIF data and overlays this data on the single photo, on all the photos

**Reply by dnewman-gpsw** [reply]
Like many tools do that using standard EXIF, which GoPro complies with.

**Comment by yorkpeter** [comment]
Then with the Startrails utility you can create either a video or a single photo that illustrates the entire phenomenon.
